requirements.txt

```diff
@@ -0,0 +1,4 @@
+ops==1.1.0
+oci-image==1.0.0
+serialized-data-interface==0.2.0
+git+https://github.com/canonical/interface-service-mesh
```

Comment:
This dependency is no longer necessary with the schema defined in `metadata.yaml`, right?